src/tests/test_magic_cmd.py

```diff
@@ -170,10 +185,18 @@ def test_columns_with_schema(ip, tmp_empty):
     assert "some_number" in out
 
 
-def test_table_profile(ip, tmp_empty):
-    ip.run_cell(
+@pytest.mark.parametrize(
+    "conn",
+    [
+        ("%sql sqlite://"),
+        ("%sql sqlite_conn"),
+    ],
```

Comment:
you can add all the connections to your fixture with an alias and then pass `%sql connectionalias` to switch to it, this is clear than that approach you're taking. also provide more clear variable names, because `sqlite_conn` doesn't immediately tell me that this is a DBAPI connection

Reply:
[fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR68-R80): Changing the fixture. Now aliases are `duckdb_sqlalchemy`, `sqlite_sqlalchemy`, `duckdb_dbapi`, and `sqlite_dbapi`.

Also [fixed](https://github.com/ploomber/jupysql/pull/785/files#diff-9188e75e824219dd191cdc62d6d19f2b15c4bd0cf564abe3646ea907c32cd61dR193-R194) parametrize arguments.